Bug 1351075: Reactivate LoginManagerTests

npark-mozilla opened the pull request:
Enable LoginManagerTests in UITests, and Move tests that sets the app to background and back to XCUITests

farhanpatel commented:
<!--
  0 Errors
  0 Warnings
  0 Messages
  1 Markdown
-->

### SwiftLint found issues

#### Warnings

File | Line | Reason |
| --- | ----- | ----- |
LoginManagerTests.swift | 493 | Limit vertical whitespace to a single empty line. Currently 2. 


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

farhanpatel commented on XCUITests/XCUITests-Bridging-Header.h:
```diff
@@ -0,0 +1,18 @@
+//
+//  XCUITests-Bridging-Header.h
```
This should be the default license

npark-mozilla replied:
@farhanpatel Ah, I added this header to enable sending the app to the background in XCUITests, like I did in https://github.com/mozilla-mobile/firefox-ios/pull/2566/files#diff-30cb5c0e8f9990cd480c6e4d3be1af8dR104 This enables to grab XCUIApplication object for a non-target apps, like springboard.  You're right about the header, I'll update that

npark-mozilla commented:
@farhanpatel Hmm, my comment got hidden. I added this header to enable sending the app to the background in XCUITests, like I did in https://github.com/mozilla-mobile/firefox-ios/pull/2566/files#diff-30cb5c0e8f9990cd480c6e4d3be1af8dR104 This enables to grab XCUIApplication object for a non-target apps, like springboard.  Also I added the header now, could you re-review?

npark-mozilla commented:
Hmm, having bb issues, contacted them to sort it out.

npark-mozilla commented:
Closing this PR and opening a new one due to BB issue
